addons/project/views/project_portal_project_task_templates.xml

```diff
@@ -270,7 +271,7 @@
     </template>
 
     <template id="portal_my_task_planned_hours_template">
-        <strong t-if="task.planned_hours > 0">Allocated Time:</strong>
+        <strong t-if="task.planned_hours > 0 and 1 == 0">Allocated Time:</strong>
```

Comment:
Does that help? If the customization refers to a tag `strong` with attribute `t-if`: `task.planned_hours > 0`, won't it break?
I think using `invisible` or class `d-none` would be preferable.

Reply:
> Does that help? If the customization refers to a tag `strong` with attribute `t-if`: `task.planned_hours > 0`, won't it break? I think using `invisible` or class `d-none` would be preferable.

done.thanks